src/vm/tests/simple_apply_eval.rs

```diff
@@ -541,6 +541,196 @@ fn test_simple_arithmetic_functions() {
         .for_each(|(program, expectation)| assert_eq!(expectation.clone(), execute(program)));
 }
 
+#[test]
+fn test_sequence_comparisons_clarity1() {
+    // Tests the sequence comparisons against ClarityVersion1. The new kinds of
+    // sequence comparison *should not* work.
+
+    // Note: Equality between sequences already works in Clarity1.
+    let success_tests = [
+        ("(is-eq \"aaa\" \"aaa\")", Value::Bool(true)),
+        ("(is-eq u\"aaa\" u\"aaa\")", Value::Bool(true)),
```

Comment:
for both the success and error tests, could add tests where the sequences are of different lengths

Reply:
I changed teh success tests from this section to use different lengths.. note that the tests in `test_sequence_comparisons_clarity2` also use different lengths.